mvlearn/embed/kcca.py

```diff
@@ -39,20 +40,25 @@ class KCCA(BaseEmbed):
     ktype : string, default = 'linear'
             Type of kernel
         - value can be 'linear', 'gaussian' or 'poly'
-    degree : float, default = 2.0
-             Degree of Polynomial kernel
     constant : float, default = 1.0
              Balances impact of lower-degree terms in Polynomial kernel
     sigma : float, default = 1.0
             Standard deviation of Gaussian kernel
+    degree : float, default = 2.0
+             Degree of Polynomial kernel
     reg : float, default = 0.1
           Regularization parameter
     decomp : string, default = 'full'
-             Decomposition type
-        - value can be only be 'full'
+             Decomposition type. Incomplete Cholesky Decomposition (ICD)
+             can reduce computation times and storage
+        - value can be 'full' or 'icd'
     method : string, default = 'kettenring-like'
              Decomposition method
         - value can be only be 'kettenring-like'
+    mrank : int, default = 2
+            The rank of the kernel matrix
```

Comment:
Make it explicit that mrank is just for ICD. maybe "The rank of the ICD approximated kernel matrix"